### PR TITLE
Update ganache chain ID

### LINKED ISSUE #592: Change the chain ID for ganache network
Change the chain ID for ganache network wherever it's hardcoded with `1337` value.

### PR DESCRIPTION
**Pull request by mariacarmina**
Fixes #592  .

Changes proposed in this PR:

- update ganache chain ID wherever is hardcoded.
- fix the failing tests after the contracts update.

### CODE REVIEW

File: ocean_lib/config.py [hunk 1]
```diff
@@ -273,7 +273,7 @@ def network_name(self) -> str:
     @property
     @enforce_types
     def chain_id(self) -> int:
-        """Chain ID of the ethereum network. (e.g.): 1337."""
+        """Chain ID of the ethereum network. (e.g.): 8996."""
```
Comment (DMats) [inline ocean_lib/config.py:276]: I recommend changing to "Chain ID of the EVM-compatible network." and removing the example.

**Review by DMats — APPROVED** [review]
I left a comment about the chain_id docstring, but overall this PR looks good.  I double-checked that all hardcoded 1337 are removed. The unit tests are failing, but that's expected for now, so I'm approving.

File: ocean_lib/config.py (hunk 1 above)
Comment (mariacarmina) [reply ocean_lib/config.py:276]: Thank you.

**Review by calina-c — APPROVED** [review]

**Comment by mariacarmina** [comment]
NOTE: After updating the artifacts folder, I still get some failing unit tests and I attempt to resolve them here.

**Comment by mariacarmina** [comment]
In the details section from the latest `build` all the tests from `models/v4/test` and `tests/flows` are passing now.

**Pull request merged**